docs/autogenerate_OAI_models.md

```diff
@@ -1,22 +1,49 @@
 ## Motivation
-You will only need to use the script to build models when any of the GA4GH API's openapi specs are updated. The models are being used to return the result of any API call in the correct format, in the correct structs
+You will only need to use the script to build models when any of the GA4GH API's OpenAPI specs are updated. The models are being used to return the result of any API call in the correct format, in the correct structs.
 
+## Prerequisites and Dependencies:
+1. The `script.sh` should have permission to run, which can be done by:
+    ```sh
+    sudo chmod +x ./utils/script.sh
```

Comment:
I think permissions are retained via Git, so if you added the script with `+x`, users won't need to do this.

Reply:
Oh, okay. I have removed it from the documentation then.